src/ServiceControl/Infrastructure/NServiceBusFactory.cs

```diff
@@ -54,16 +55,12 @@ public static Task<IStartableEndpoint> Create(Settings.Settings settings, Transp
 
             if (!isRunningAcceptanceTests)
             {
-                configuration.ReportCustomChecksTo(endpointName);
+                configuration.EnableFeature<InternalCustomChecks>();
             }
 
             configuration.UseContainer<AutofacBuilder>(c => c.ExistingLifetimeScope(container));
 
-            configuration.DefineCriticalErrorAction(criticalErrorContext =>
-            {
-                onCriticalError(criticalErrorContext);
```

Comment:
We are still passing this in but it doesn't get called anywhere. For instance, it was once used to shut down an acceptance test on critical failure. I'm guessing we no longer need to do that and can rely on the test timeouts to shut the ATT down.

Reply:
Yeah. I'll remove it.